kitchen.yml

```
@@ -33,7 +33,7 @@ suites:
     run_list:
       - recipe[test::distro_nginx-full]
       - recipe[test::test_site]
-    includes: # this may be platform specific
+    includes: #  this may be platform specific
```

Actually, I was wrong. I believe this is correct fix.
```suggestion
    includes:  # this may be platform specific
```